### ISSUE TITLE
Reroll summon bug in gw mode (TreasureEventMode)

### ISSUE THREAD

**Issue by Dreamfinal** [Masuzu/ZooeyBot#131]
When enter treasureeventmode (gw) , the program didn't select the target summon. But randomly select the light summon instead.

```
> [Summons]
> PreferredSummons=Lucifer_2
> DefaultSummonAttributeTab=Light
> RerollSummonWhenNoPreferredSummonWasFound=true
```

**Comment by Masuzu** [comment]
With these settings, If the summon is not found, then it will select a random summon and proceed to reset the summon list using the trial mode.
Or maybe you are refering to a similar issue as the second bullet point of the patch notes of the version 1.10.0: https://poker-bot.azurewebsites.net/ZooeyBot/en/Home/PatchNotes?

**Comment by Ziragat** [comment]
Im getting the same error, when set to reroll summon for the nm90 if the select summon is not available it picks any light summon instead of rerollin. 
Works fine for event farming tho, its an issue with the nightmare

**Comment by Masuzu** [comment]
Ah my bad, I see. Thank you for the clarification.

**Comment by Masuzu** [comment]
This should be fixed in version 1.11.0, let me know if the issue still persists.
